hyperdrive/packages/hypermap-cacher/hypermap-cacher/src/lib.rs

```diff
@@ -146,7 +159,11 @@ impl State {
             is_cache_timer_live: false,
             drive_path: drive_path.to_string(),
             is_providing: false,
-            nodes: DEFAULT_NODES.iter().map(|s| s.to_string()).collect(),
+            nodes: load_cache_sources(std::env::var("CACHE_SOURCE_CONFIG_PATH").ok().as_deref())
```

Comment:
I'd be surprised if this works: I think we need to do some special things when starting a process for it to see env vars